chain/network/src/network_protocol/mod.rs

```diff
@@ -411,6 +411,8 @@ pub enum RoutedMessageBody {
 
     StateRequestHeader(ShardId, CryptoHash),
     StateRequestPart(ShardId, CryptoHash, u64),
+    /// StateResponse in not produced since protocol version 58.
+    /// We can remove the support for it in protocol version 60.
```

Comment:
mention that we use VersionedStateResponse instead.

Reply:
done